src/main/kotlin/org/opensearch/commons/alerting/model/Comment.kt

```diff
@@ -19,16 +19,17 @@ import java.time.Instant
 data class Comment(
     val id: String = NO_ID,
```

Comment:
could you add Writeable & XContent tests for comment model?
https://github.com/opensearch-project/common-utils/blob/main/src/test/kotlin/org/opensearch/commons/alerting/model/WriteableTests.kt
https://github.com/opensearch-project/common-utils/blob/main/src/test/kotlin/org/opensearch/commons/alerting/model/XContentTests.kt

Reply:
I'll add in this PR to avoid merge conflicts: https://github.com/opensearch-project/common-utils/pull/678